index.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Microverse Project by Chris</title>
+    <link rel="stylesheet" href="assets/css/style.css">
+    <link rel="stylesheet" href="assets/css/desktop.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+</head>
+<body>
+  <header>
+    <nav>
+      <a href="/">My Logo</a>
+      <button onclick="return;" type="button">
+        <span>
+          <img
+            src="assets/images/icons/Union.png" 
+            alt="nav button">
+        </span>
+      </button>
+    </nav>
+  </header>
+  <main>
+    <div class="backgrounds">
+      <img class="ellipse11"
+        src="assets/images/icons/Ellipse 11.png"
+        alt="background icon">
+      <img class="ellipse8"
+        src="assets/images/icons/Ellipse 8.png"
+        alt="background icon">
+      <img class="ellipse5"
+        src="assets/images/icons/Ellipse 5.png"
+        alt="background icon">
+      <img class="rect7"
+        src="assets/images/icons/Rectangle 7.png"
+        alt="background icon">
+      <img class="ellipse12"
+        src="assets/images/icons/Ellipse 12.png"
+        alt="background icon">
+      <img class="ellipse13"
+        src="assets/images/icons/Ellipse 13.png"
+        alt="background icon">
+      <img class="ellipse14"
+        src="assets/images/icons/Ellipse 14.png"
+        alt="background icon">
+      <img class="ellipse15"
+        src="assets/images/icons/Ellipse 15.png"
+        alt="background icon">
+      <img class="ellipse3"
+        src="assets/images/icons/Ellipse 3.png"
+        alt="background icon">
+      <img class="ellipse4"
+        src="assets/images/icons/Ellipse 4.png"
+        alt="background icon">
+      <img class="rect4"
+        src="assets/images/icons/Rectangle 4.png"
+        alt="background icon">
+      <img class="rect5"
+        src="assets/images/icons/Rectangle 5.png"
+        alt="background icon">
+      <img class="rect6"
+        src="assets/images/icons/Rectangle 6.png"
+        alt="background icon">
+    </div>
```

Comment:
- [ ] The images you have used here are background images. It is the best practice to use `background-image` property in CSS for background images. Using HTML `img` tag for background images is not a good practice. Kindly use `background-image` property in CSS file. You can consider going through this [link](https://www.w3schools.com/cssref/pr_background-image.php).
- [ ] It would be better to download the shapes of background images from the [template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) altogether instead of using one by one shapes. As it would be easy to maintain.

- Here is an example of how to download the shapes altogether:
- ![ezgif com-video-to-gif (1)](https://github.com/chudisoft/Portfolio-Microverse/assets/60083437/aaed14ce-6689-43d9-b1bf-c3a684b73fdc)

Reply:
Done